.github/workflows/presubmit.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: 'tikurahul/check-pr-format-action@master'
         with:
           # Enforces a `Test: <stanza>` in the pull request
-          checks: '["(.*)?Test\:(.*)?"]'
```

Comment:
Did you mean to revert this?

Reply:
No, this was a stacked change. I will rebase now.